src/encoding.md

```diff
@@ -344,6 +345,60 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-invalid-csv]: #ex-invalid-csv
+<a name="ex-invalid-csv"></a>
+## Handle invalid CSV data with Serde
+
+[![csv-badge]][csv] [![serde-badge]][serde] [![cat-encoding-badge]][cat-encoding]
+
+We get deserialization errors if [`invalid CSV data`] is encountered. [`Serde`] is
+essential to convert any such errors to a [`None`] value, when applied to
+the [`Option`] field.
```

Comment:
The second sentence here, "Serde is essential to convert any errors ..." seems off to me. This examples looks to be about using `csv::invalid_option` to deal with bogus CSV data.

I might instead word this as something like

> CSV files often contain invalid data. For these cases the `csv` crate provides a custom deserializer, `csv::invalid_option`, which automatically converts invalid data to `None` values.

Place a link on `csv::invalid_option`, but remove the links from `Some` and `None` - those are very fundamental types that don't need links here IMO.